pages/linux/arp-scan.md

```diff
@@ -0,0 +1,19 @@
+# arp-scan
+
+> Send ARP packets to hosts on the local network to scan which ones answer.
+
+- Scan the current local network:
+
+`arp-scan --localnet`
+
+- Scan an IP network with a custom bits mask:
+
+`arp-scan {{192.168.1.1}}/{{24}}`
+
+- Scan an IP network within a custom range:
+
+`arp-scan {{127.0.0.0}}-{{127.0.0.31}}`
+
+- Scan an IN network with a custom net mask:
```

Comment:
What's the difference between an `IP` network and an `IN` network?

Reply:
The difference is a typo, my mistake